email_service/email_lambda.py

```diff
@@ -36,7 +70,18 @@ def __init__(self) -> None:
         """
         self.logger = logger
 
-    def get_user_data(self, conn, data):
+    def get_connection(self) -> connection:
```

Comment:
DRY.

We do not need to create this again, I can show you how to import from siblings dirs tomorrow.